src/shipit_code_coverage/tests/test_github.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+import responses
+
+from shipit_code_coverage.github import GitHubUtils
+
+
+@responses.activate
+def test_get_commit(MERCURIAL_COMMIT, MERCURIAL_COMMIT_RESPONSE):
+    responses.add(
```

Comment:
I meant that you can put this entire function in a fixture (here and for `test_get_mercurial`)